src/browser/browser.js

```diff
@@ -255,8 +258,22 @@ define((require, exports, module) => {
         onClose: item => webViewersCursor.update(closeTab(item)),
         onOpen: item => webViewersCursor.update(addTab(item))
       })
-    ]);
-  });
+    ]),
+    DOM.div({
```

Comment:
Oh we can absolutely create the element and animate once mounted, but we can do that later.